### ISSUE TITLE
Support for AllowActiveFTP parameter on Azure Firewall

### ISSUE THREAD

**Issue by tejasshah7** [Azure/azure-cli#14369]
**Resource Provider**
Network (NRP)

**Description of Feature or Work Requested**
Need to add a new parameter named AllowActiveFTP to the base command of creating the Firewall.
az network firewall create -g MyResourceGroup -n MyFirewall --allow-active-ftp true

Only parameter needed is:
- allow-active-ftp
Default value is false

**Minimum API Version Required**
There is no NRP change so no API version issue.

**Swagger Link**
There is no swagger change.

**Target Date**
End of July 2020

**Comment by yonzhan** [comment]
add to S174

**Comment by tejasshah7** [comment]
This is not needed at this time. Sorry about submitting it.

**Comment by tejasshah7** [comment]
I just found that we may still need this. I had a quick question. If today someone creates this firewall with this new property and then does a get via az cli to modify some other parameter and does a put on the firewall without this new property, will it be lost? If so, then we would need this.

**Comment by MyronFanQiu** [comment]
@tejasshah7 From server side, you won't receive this field at all if you don't set any default value in swagger spec. I assume if server doesn't receive this value, it won't be changed.

**Comment by tejasshah7** [comment]
@MyronFanQiu server will return this value if it is set from ARM and is NOT a default value. In that case, if a Put is done from az cli will it carry the field back or not? If not, then it will reset it in the backend and we would need the support in az cli for it.
Powershell has this issue and hence I am going to add support for this in Powershell.

**Comment by MyronFanQiu** [comment]
@tejasshah7 For an update scenario, if server return this field and you also define this field in swagger spec, az cli would put same value back to server right now.

**Comment by MyronFanQiu** [comment]
I cannot find this field in the swagger spec. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/azureFirewall.json In that case, CLI would drop this field and won't send it back.

**Comment by tejasshah7** [comment]
Exactly, This field is not in swagger. Then we will need support for this feature.

**Comment by MyronFanQiu** [comment]
@tejasshah7 I assume there will be a swagger PR soon?

**Comment by tejasshah7** [comment]
There is no swagger change for this. These fields are like:
https://github.com/Azure/azure-cli/issues/13350

**Comment by MyronFanQiu** [comment]
well... If user doesn't set this field, should server keep it same instead of reset it? Please provide enough examples and schema for this feature request if you don't have swagger spec.

**Comment by tejasshah7** [comment]
I am talking about the scenario where user sets this field say using Powershell. Here is the powershell PR for reference:
https://github.com/Azure/azure-powershell/pull/12430

**Comment by tejasshah7** [comment]
@yonzhan  what is the status for this?

**Comment by MyronFanQiu** [comment]
@tejasshah7 This feature is under development. There will be a PR in the earlier next week. We still don't recommend using additional property like this. It's like a black box for us.
